samples/tab-deeplink/nodejs/README.md

```diff
@@ -64,6 +64,9 @@ This sample displays how to consume SubEntity Id to [DeepLink](https://docs.micr
         npm install
         ```
 
```

Comment:
Let's raise another PR to remove another sample

Reply:
PR link to remove the sample
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/723